content/en/getting_started/agent.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Getting Started with the Agent
+kind: documentation
+further_reading:
+- link: "/agent/basic_agent_usage/"
+  tag: "Documentation"
+  text: "Basic Agent Usage"
+- link: "/agent/faq/why-should-i-install-the-agent-on-my-cloud-instances/"
+  tag: "FAQ"
+  text: "Why should I install the Datadog Agent on my cloud instances?"
+---
+
+## Overview
+The Agent is lightweight software installed on your hosts. It reports metrics and events from your host to Datadog. With additional setup, the Agent can report [live processes][1], [logs][2], and [traces][3].
```

Comment:
Since this is an introductory overview, this might be a really good place for a short definition of what an agent actually is generally, and maybe a diagram about how the agent is collecting information and sending it to Datadog. It's also worth pointing out (and mostly linking out) to security information about exactly which data we're collecting and why it's secure.

Reply:
The Agent page (https://docs.datadoghq.com/agent) has more in-depth information about the Agent. This page is just meant to be the "Get me started quick page".